sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/UnsupportedOperationChecker.scala

```diff
@@ -417,7 +417,7 @@ object UnsupportedOperationChecker extends Logging {
 
       subPlan.expressions.foreach { e =>
         if (e.collectLeaves().exists {
-          case (_: CurrentTimestamp | _: Now | _: CurrentDate) => true
+          case (_: CurrentTimestamp | _: Now | _: CurrentDate | _: LocalTimestamp) => true
```

Comment:
Let's just check `CurrentTimestampLike` instead of `CurrentTimestamp`, `Now` and `LocalTimestamp `